### PR TITLE
Implemented StampBoard 3d component and board example

### PR DESCRIPTION
**Pull request by anas-sarkez**
**With holes**

Screenshot 2024-11-16 at 7 32 46 AM

**without holes**

Screenshot 2024-11-16 at 8 09 46 AM

/claim #79

### CODE REVIEW

**Review by seveibar — APPROVED** [review]
Can we rename this to StampBoard before merging?

**Comment by anas-sarkez** [comment]
> Can we rename this to StampBoard before merging?

@seveibar 
what I should rename the pr of example file?

File: lib/stampboard.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
+import { Colorize, Cuboid, Cylinder, Subtract, Union } from "jscad-fiber"
+
+export const Board = ({
```
Comment (seveibar) [inline lib/stampboard.tsx:3]:
```suggestion
export const StampBoard = ({
```

**Review by seveibar — COMMENTED** [review]
stampboard.example.tsx as well. When it comes to 3d models, there is no stamp receiver. A Stamp receiver footprint is what we call the pads that you glue a stampboard to, and a stampboard footprint is what we call the pads that are on a stampboard

The receiver footprint has longer pads, otherwise it wont be able to glue to stampboard as well

**Pull request merged**